src/core/algorithms/md/hymd/similarity_data.cpp

```diff
@@ -5,6 +5,20 @@
 #include "algorithms/md/hymd/indexes/column_similarity_info.h"
 #include "algorithms/md/hymd/lowest_cc_value_id.h"
 
+namespace {
+struct IndexInfo {
+    std::size_t lhs_to_rhs_size;
+    std::size_t rhs_to_lhs_size;
```

Comment:
```suggestion
    std::size_t lhs_ccv_id_number;
    std::size_t rhs_ccv_id_number;
```
Also check the names in the for-each loops.